libcloud/compute/drivers/vultr.py

```diff
@@ -119,6 +133,15 @@ def create_node(self, name, size, image, location):
 
         return created_node
 
+    def ex_list_ssh_keys(self):
```

Comment:
Also, looking at the docs at http://search.cpan.org/~eskaaren/WebService-Vultr-0.1/lib/WebService/Vultr.pm, it looks like vultr also returns public key so you can also set that attribute.

```
Example Response: { "541b4960f23bd": { "SSHKEYID": "541b4960f23bd", "date_created": null, "name": "test", "ssh_key": "ssh-rsa AA... test@example.com" } }
```